sambaplayersdk/src/main/java/com/sambatech/player/mediasource/PlayerInstanceDefault.java

```diff
@@ -29,13 +36,37 @@
     Context context;
     DataSource.Factory mediaDataSourceFactory;
 
-    public PlayerInstanceDefault(Context context) {
+    private HttpMediaDrmCallback drmCallback;
+    private DefaultDrmSessionManager drmSessionManager;
+
+    public PlayerInstanceDefault(Context context, SambaMediaConfig media) {
+
+        boolean isDRM = media.drmRequest != null;
+
         this.context = context;
         this.mainHandler = new Handler();
         this.bandwidthMeter = new DefaultBandwidthMeter();
         this.adaptiveTrackSelectionFactory =  new AdaptiveTrackSelection.Factory(bandwidthMeter);
         this.trackSelector = new DefaultTrackSelector(adaptiveTrackSelectionFactory);
-        this.renderersFactory = new DefaultRenderersFactory(this.context);
+
+        if (isDRM) {
+            drmCallback = new HttpMediaDrmCallback(media.drmRequest.getLicenseUrl(), new DefaultHttpDataSourceFactory("user-agent"));
+            if (media.drmRequest.getProvider() != null && media.drmRequest.getProvider().equals("SAMBA_DRM")) {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Move the "SAMBA_DRM" string literal on the left side of this string comparison. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar.sambatech.com/coding_rules#rule_key=squid%3AS1132)